### PR TITLE
Automation request rest api

### PR DESCRIPTION
**Pull request by lehinevych**
{{pytest: cfme/tests/infrastructure/test_rest_automation_request.py -v --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit c2ff85191d09cf02304259c5fd283b2553c5c382:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c5881223295b7006679fb7a5268efb858e07710e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/rest/__init__.py [hunk 1]
```diff
@@ -297,3 +297,22 @@ def _finished():
...
+        "uri_parts": {
+            "namespace": "System",
+            "class": "Request",
+            "instance": "InspectME",
+            "message": "create",
+        },
+        "parameters": {
+            "vm_name": "test_rest_{}".format(fauxfactory.gen_alphanumeric()),
+            "vm_memory": 4096,
+            "memory_limit": 16384,
```
Comment (mfalesni) [inline cfme/rest/__init__.py]: Looking at this, you sure that this is entirely correct? InspectMe expects some DB object passed, like VM, or Host, this is not a provisioning request. I know this is just moving of data but I haven't spotted this before :)

**Comment by dajoRH** [comment]
Lint report for commit 5e68cc7a2827cc564f222197f5af4ae969a9b8cc:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 208fbf3d97cd1bf77befbfbf4ee0d32cccdac515:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by lehinevych** [comment]
@mfalesni Now existing vm is passed, so it should be ok for InspectMe. Am I right or missing something?

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3ddf9a36df4254776b35f8e46b5c0a6f8c65cbbd:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**